systemtests/tests/messages-resource/etc/bareos/bareos-dir.d/messages/Standard.conf.in

```diff
@@ -2,7 +2,7 @@ Messages {
   Name = Standard
   Description = "Message delivery for all reasonable directions"
   Console = all
-  Append = "@logdir@/append" = all
+  Append = "@logdir@/append-to-a-very-long-file-12345678901234567890123456789012345678901234567890abcdefghijklmnopqrstuvwxyz1234567890" = all
```

Comment:
this is only 115 chars + @logdir@ which could end up with less than 127 chars.

Reply:
done